### PR TITLE
"8.x-2.x >  8.2.1

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- Automated pull request created via HAL 9000.  See changes below for details.

# Review By (Date)
- As soon as possible

The following changes since commit 1baa0ec932e3bbdb4eb4b370e46609d1d46a5bf6:

  8.2.0 (2020-05-15 11:16:08 -0600)

are available in the Git repository at:

  ./ 

for you to fetch changes up to 56b31d93d825b90476007c979fea8219688365be:

  8.2.1 (2020-05-20 12:53:39 -0600)

----------------------------------------------------------------
Ian Monroe (2):
      back to dev (#77)
      8.2.1

Sherakama (2):
      D8CORE-2110: Removed news views paragraph type. (#78)
      D8CORE-000: Changed headline field to 180 characters. (#79)

pookmish (1):
      Back to dev for su-sws composer packages

 CHANGELOG.md                                       |  7 ++
 composer.json                                      |  7 +-
 ...y_form_display.paragraph.news_views.default.yml | 22 ------
 ...y_view_display.paragraph.news_views.default.yml | 24 ------
 ...d.field.node.stanford_news.su_news_headline.yml |  2 +-
 ...eld.paragraph.news_views.su_news_view_field.yml | 40 ----------
 .../field.storage.node.su_news_headline.yml        |  2 +-
 .../field.storage.paragraph.su_news_view_field.yml | 20 -----
 .../paragraphs.paragraphs_type.news_views.yml      | 11 ---
 stanford_news.info.yml                             | 86 +++++++++++-----------
 stanford_news.module                               | 32 --------
 11 files changed, 53 insertions(+), 200 deletions(-)
 delete mode 100644 config/install/core.entity_form_display.paragraph.news_views.default.yml
 delete mode 100644 config/install/core.entity_view_display.paragraph.news_views.default.yml
 delete mode 100644 config/install/field.field.paragraph.news_views.su_news_view_field.yml
 delete mode 100644 config/install/field.storage.paragraph.su_news_view_field.yml
 delete mode 100644 config/install/paragraphs.paragraphs_type.news_views.yml"

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -89,13 +89,12 @@
...
-        "drupal/taxonomy_menu": "dev-3.x",
-        "drupal/viewfield": "^3.0-beta5",
+        "drupal/taxonomy_menu": "dev-3.x#c0892fa9b428ceb784876554c9b80f49d72a246f",
         "drupal/views_block_filter_block": "^1.0",
         "drupal/views_infinite_scroll": "^1.6",
         "drupal/views_taxonomy_term_name_depth": "^6.0",
-        "su-sws/jumpstart_ui": "^8.1.5",
-        "su-sws/stanford_text_editor": "^8.1.6"
+        "su-sws/jumpstart_ui": "dev-8.x-1.x",
+        "su-sws/stanford_text_editor": "dev-8.x-1.x"
```
Comment (sherakama) [inline composer.json:97]: @imonroe These need to go back to tags correct?

File: stanford_news.info.yml [hunk 2]
```diff
@@ -1,51 +1,47 @@
...
-  - ui_patterns:ui_patterns
-  - ui_patterns:ui_patterns_layouts
-  - ui_patterns:ui_patterns_library
-  - ui_patterns:ui_patterns_views
-  - viewfield:viewfield
-  - views_block_filter_block:views_block_filter_block
-  - views_infinite_scroll:views_infinite_scroll
-  - views_taxonomy_term_name_depth:views_taxonomy_term_name_depth
-
+    - 'allowed_formats:allowed_formats'
```
Comment (sherakama) [inline stanford_news.info.yml:8]: This don't need to be wrapped in quotes.

File: stanford_news.info.yml [hunk 3]
```diff
@@ -1,51 +1,47 @@
...
+    - 'ui_patterns:ui_patterns_library'
+    - 'ui_patterns:ui_patterns_views'
+    - 'views_block_filter_block:views_block_filter_block'
+    - 'views_infinite_scroll:views_infinite_scroll'
+    - 'views_taxonomy_term_name_depth:views_taxonomy_term_name_depth'
 component-libraries:
-  news:
-    paths:
-      - templates/components
+    news: { paths: [templates/components] }
```
Comment (sherakama) [inline stanford_news.info.yml:47]: Why the change to the format?

Comment (imonroe) [reply stanford_news.info.yml:47]: That's not a change I made manually; the php YAML library does that for some reason.  It's not necessary.

File: stanford_news.info.yml (hunk 2 above)
Comment (imonroe) [reply stanford_news.info.yml:8]: This is another YAML object thing.  I need to build in some linting so it follows our preferred style.

File: composer.json (hunk 1 above)
Comment (imonroe) [reply composer.json]: Correct.  Fixed it.

File: stanford_news.info.yml (hunk 3 above)
Comment (sherakama) [reply stanford_news.info.yml:47]: Ah, I figured it had to do with yaml parsing and outputting.

File: stanford_news.info.yml [hunk 4]
```diff
@@ -1,51 +1,47 @@
 name: 'Stanford News'
 type: module
 description: 'Stanford news items and views.'
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: '^8.8 || ^9'
```
Comment (sherakama) [inline stanford_news.info.yml:4]: No quotes.

File: stanford_news.info.yml (hunk 2 above)
Comment (imonroe) [reply stanford_news.info.yml]: fixed it.

File: stanford_news.info.yml (hunk 3 above)
Comment (imonroe) [reply stanford_news.info.yml]: fixed it.

File: stanford_news.info.yml (hunk 4 above)
Comment (imonroe) [reply stanford_news.info.yml]: fixed it.
